README.md

```diff
@@ -56,7 +67,13 @@ be given with `DSSC_IMAGE_NAME`.
       "username": "<user>",
       "password": "<password>"
     }
+    ``` 
+  - If you're using AWS, you can use this example below: 
```

Comment:
I was unable to get this action to work for GCR. It might have had to do with how I specified `DSSC_IMAGE_PULL_AUTH `, but I was unable to figure out how to specify this variable even after following the link below. If this is possible we should link to a resource which describes how to do it, otherwise we should mention in the README what subset of registry-types the Github action is able to handle.

Reply:
Let me write something to cover GCR as well, I think will be good to add that...